test/test_grammars.rb

```diff
@@ -5,45 +5,36 @@ class TestGrammars < Minitest::Test
+    "6c2e34d62c08f97a3e2ece3eedc65fbd99873ff4", # idl.tmbundle
+    "e5212ae103917a9c2c3c1429a4569df466686fbd", # Isabelle.tmbundle
+    "bb56ce634fb7ddd38eee988c593ab7cb98a04f64", # jflex.tmbundle
+    "41cdc7e9f9d2e62eb8ac68a1a9359b9c39a7a9bf", # mako-tmbundle
+    "7821982b18bc35d6925cc16ece68d9c71f1fbba3", # moonscript-tmbundle
+    "c235154dbf7864612ac0d337ef5fe79a586b061a", # PHP-Twig.tmbundle
+    "0c216b112f3a4e6d5848128504d8378d8c7eee00", # r.tmbundle
+    "44ae3474f0d3d037984c7e8d52adc4d3eb39a200", # RDoc.tmbundle
     "da39a3ee5e6b4b0d3255bfef95601890afd80709", # SCSS.tmbundle
-    "b5432a1e1055de7eeede2dddf91e009480651fd6", # jasmin-sublime
-    "170b35df61879139b88379a8f1bfd86289c13599", # language-clojure
-    "60e1fe192238a032341d5dd3cd80535459fc84e4", # language-coffee-script
-    "94fbd554ec1837fb7c508fd7425326639c3f4103", # language-csharp
-    "70fb557a431891c2d634c33fa7367feab5066fd6", # language-javascript
-    "8653305b358375d0fced85dc24793b99919b11ef", # language-shellscript
-    "9f0c0b0926a18f5038e455e8df60221125fc3111", # elixir-tmbundle
-    "a4dadb2374282098c5b8b14df308906f5347d79a", # mako-tmbundle
-    "e06722add999e7428048abcc067cd85f1f7ca71c", # r.tmbundle
-    "50b14a0e3f03d7ca754dac42ffb33302b5882b78", # smalltalk-tmbundle
-    "eafbc4a2f283752858e6908907f3c0c90188785b", # gap-tmbundle
-    "22b3bf41b9e3e8c22357ee12265f149d68aae60a", # Stylus
-    "c87e7e574fca543941650e5b0a144b44c02c55d8", # language-crystal
-    "ace112feb693358db2970d0805f6894b745e14b5", # atom-language-purescript
-    "a626362e3efd030c1d97c0faf422cf8c2dfaea54", # FreeMarker.tmbundle
-    "15a394f6bc43400946570b299aee8ae264a1e3ff", # language-renpy
-    "74bb588102e8f332970a0fcabe36299e0806f130", # language-less
-    "2f03492b52d7dd83b4e7472f01b87c6121e5b1a4", # monkey
-    "784da5ce445892bc3e26beeb6a4402bbc5ca997e", # ant.tmbundle
-    "bdab9fdc21e6790b479ccb5945b78bc0f6ce2493", # language-blade
-    "c9118c370411f2f049c746c0fd096554e877aea2", # atom-language-perl6
-    "15a502335012f27f8a5991139298edb87a6e467d", # atom-language-rust
-    "304be6184f7f344d44a1d13bddf511019624fd22", # language-css
-    "8c538244ba88ef9902a4faf11a2b9acec46f2a4e", # sublime-nginx
-    "82c356d6ecb143a8a20e1658b0d6a2d77ea8126f", # idl.tmbundle
-    "9dafd4e2a79cb13a6793b93877a254bc4d351e74", # sublime-text-ox
-    "8e111741d97ba2e27b3d18a309d426b4a37e604f", # sublime-varnish
+    "68539730d3cde34355f429f2267e265c1e030912", # smalltalk-tmbundle
+    "4b5f67a54532ca6e49ba44cd135a510a74712e07", # Stylus
     "23d2538e33ce62d58abda2c039364b92f64ea6bc", # sublime-angelscript
-    "53714285caad3c480ebd248c490509695d10404b", # atom-language-julia
+    "966085b715baa0b0b67b40924123f92f90acd0ba", # sublime-shen
+    "3df4ef028c6384b64bc59b8861d6c52093b2116d", # sublime-text-ox
+    "fd47e09f1fbdb3c26e2960d0aa2b8535bbc31188", # sublimetext-cuda-cpp
+    "93360925b1805be2b3f0a18e207649fcb524b991", # Std license in README.md of many TextMate grammars like abap.tmbundle
```

Comment:
All these changes are because Licensee now strips out "All rights reserved" text and markup before calculating the hash. This results in better license detection and also detection of identical licenses in the README of a lot of the TextMate grammars like that in abap.tmbundle. It also means that come of the old entries that remain now have a new hash.